tap_freshdesk/client.py

```diff
@@ -0,0 +1,87 @@
+import time
+import collections
+import functools
+import backoff
+import requests
+import singer
+
+
+LOGGER = singer.get_logger()
+BASE_URL = "https://{}.freshdesk.com"
+
+
+def ratelimit(limit, every):
```

Comment:
This function is already present under singer-python, so please import it and use it directly. 
Reference: https://github.com/singer-io/singer-python/blob/master/singer/utils.py#L81-L99

Reply:
Removed the defined function from the tap and used the pre-defined singer functions for rate limit